models/profileDiffs.js

```diff
@@ -0,0 +1,49 @@
+const firestore = require("../utils/firestore");
+const profileDiffsModel = firestore.collection("profileDiffs");
+
+/**
+ * Add profileDiff
+ *
+ * @param profileDiffData { Object }: Data to be added
+ */
+const add = async (profileDiffData) => {
+  try {
+    const profileDiff = await profileDiffsModel.add({
+      ...profileDiffData,
+    });
+    return profileDiff.id;
+  } catch (err) {
+    logger.error("Error in adding profile diff", err);
+    throw err;
+  }
+};
+
+/**
+ * Update profileDiff
+ *
+ * @param profileDiffData { Object }: Data to be added
+ * @param profileId { String }: Id of the profileDiff
+ */
+
+const update = async (profileDiffData, profileId) => {
```

Comment:
`update` is a very common word... Could we be more specific here please

Reply:
Is `updateProfileDiff` ok?